src/lib.rs

```diff
@@ -32,6 +32,7 @@ pub mod controller;
 mod decorations;
 mod diff;
 pub mod error;
+pub mod ignored_suffixes;
```

Comment:
I would like to be very conservative with introducing new child modules to the public API. Can we rearrange the code somehow to avoid the need to do that?

Reply:
I managed to address your concern with this change. https://github.com/sharkdp/bat/pull/1892/commits/b311b7e949cc981f467ccf075470365339e62f7f
I hope it doesn't bring other issues - hope you are fine with modifying `SyntaxMapping::builtin` method.